manifest.json

```diff
@@ -2561,6 +2634,52 @@
                 }
             }
         },
+        "Banane9.AvatarClothingHelper": {
+            "name": "AvatarClothingHelper",
+            "description": "Automatically sets up Blendshapes with the same name on multiple renderers to be driven by the "primary" mesh's values on import or when pressing a button on the SkinnedMeshRenderer.",
```

Comment:
Invalid JSON, double quotes need to be escaped.

Reply:
Oop, I missed something... glad it's only that after merging json by hand :'D